### ISSUE TITLE
OAuth full access.

### ISSUE THREAD

**Issue by Carreau** [lemurheavy/coveralls-public#422]
Since Yesterday, trying to access my page on coveralls.io redirect me and ask your org full access to my repositories, while before only coveralls user/org was authorized with a much smaller scope. 

Is this normal ?

**Comment by nickmerwin** [comment]
Hi @Carreau, we ask for private access when we detect there's an active subscription for an organization you have access to -- it would allow you access to those private repo(s). Of course it's up to you if you want to keep public OAuth creds.

Thanks!

**Comment by Carreau** [comment]
Hum that's weird; because non of the organisation I belong too had subscription. 
Was a little annoying because I really was not able to access coveralls without granting access, or using separate browser where I was not logged in.

Anyway, I flushed all caches agressively, revoked most app permission on github an retry. Seem to work correctly again and do not ask me. 

Thanks !

**Comment by MartinNowak** [comment]
Same problem here, cleaning cookies and revoking access fixed the issue.
This happened after extending the existing OAuth with `commit status` access, it kept asking me for full private access.

**Comment by nickmerwin** [comment]
hey @MartinNowak could you post a screenshot of the GH auth page you're seeing?

**Comment by MartinNowak** [comment]
Sorry can't, but here is what happened. I didn't login on coveralls.io for quite a while, so it asked me for the to grant a new privilege `commit status`. On the GH page it was also showing me the organization I'm part of, saying something like "this organization also granted rights" and "organizations decide themselves what to allow".
After that I was logged in and saw my normal "YOUR REPOSITORIES" overview. Once I clicked on one of the repos I was redirected to the GH Auth page again. This time asking me for full access.
After that it insisted on doing so even after I logged and also for the main page.
